Makefile

```diff
@@ -45,8 +66,46 @@ cockpit/download: Makefile
 
 .PHONY: cockpit/build
 cockpit/build: cockpit/download
-	npm ci
 	npm run build:cockpit
 
 .PHONY: cockpit/devel
 cockpit/devel: cockpit/devel-uninstall cockpit/build cockpit/devel-install
+
+#
+# Building packages
+#
+
+RPM_SPEC=cockpit/$(PACKAGE_NAME).spec
+NODE_MODULES_TEST=package-lock.json
+RPM_NAME=cockpit-composer
```

Comment:
since rpm_name and package_name are the same now, we might as well use package_name everywhere I think.